mods/ts/rules/trees.yaml

```diff
@@ -113,3 +125,78 @@ VEINHOLE:
 		Palette: player
 	WithSpriteBody:
 	AutoSelectionSize:
+
+FONA01:
+	Inherits: ^Tree
+	Tooltip:
+		Name: Fona
```

Comment:
Please use inheritance instead of repeating this 15 times.  Lets please also give it a better name if possible.